flash/video/classification/input_transform.py

```diff
@@ -48,7 +52,8 @@ def per_sample_transform(self) -> Callable:
             per_sample_transform = [CenterCrop(self.image_size)]
 
         return ApplyToKeys(
-            "video", Compose([UniformTemporalSubsample(self.temporal_sub_sample)] + per_sample_transform)
+            "video",
+            Compose([UniformTemporalSubsample(self.temporal_sub_sample), Lambda(lambda_tmp)] + per_sample_transform),
```

Comment:
I think the `Lambda` may not be needed here as you could just use the function directly:
```suggestion
            Compose([UniformTemporalSubsample(self.temporal_sub_sample), normalize] + per_sample_transform),
```